thoth/metrics_exporter/jobs/db.py

```diff
@@ -24,6 +24,7 @@
 from .base import register_metric_job
 from .base import MetricsBase
 from ..configuration import Configuration
+from thoth.common import format_datetime
```

Comment:
I think the format from thoth-storages query (2022-01-31 06:31:21.345953)  is different from the one required in thoth-common https://github.com/thoth-station/common/blob/5c7452c5140a0105acfa6260ffa244d84c1f7431/thoth/common/helpers.py#L39

Reply:
> Was there also a new release with the linked change?

I just created a new minor release https://github.com/thoth-station/storages/pull/2598

Reply:
The result of the query is a `datetime` object which is taken by `format_datetime` to output a `str` with the format `"%Y-%m-%dT%H:%M:%S.%f"`. As long as the metric `"datetime"` label can be set to this value, I think that the format is correct?